documentation/src/docs/streamcuts.md

```diff
@@ -91,4 +91,4 @@ fetch segments which reside between the given `startStreamCut` and `endStreamCut
 It must be noted that passing ```StreamCut.UNBOUNDED``` to `startStreamCut` and `endStreamCut` will result in using the current head of stream and the current tail of the stream, respectively.
 
 
-We have provided a simple yet illustrative example of using StreamCut [here](https://github.com/pravega/pravega-samples/tree/v0.3.2/pravega-client-examples/src/main/java/io/pravega/example/streamcuts).
```

Comment:
The new apis to obtain StreamCuts needs to be added to the documentation.

Reply:
Addressed Your comments, Please check!